resharper/resharper-unity/src/Settings/UnitySettings.cs

```diff
@@ -26,5 +26,13 @@ public class UnitySettings
 
         [SettingsEntry(GutterIconMode.CodeInsightDisabled, "Unity highlighter scheme for editor.")]
         public GutterIconMode GutterIconMode;
+
+        
+        // TODO : merge this two options : any converter for settings?
+        [SettingsEntry(true, "Enables heuristic which can disable yaml for performance reason")]
+        public bool EnableYamlHeuristic;
+        
+        [SettingsEntry(true, "Enables syntax error highlighting, brace matching and more of YAML files for Unity")]
+        public bool EnableYamlParsing;
```

Comment:
Do we need both of these? `EnableYamlParsing` sounds just like the one in `YamlSupport`. I thought we only needed another setting with a three state - enabled (default), heuristically disabled and force enabled.

Reply:
I wanted to merge it, but don't find a good solution. 
There is problem with checkbox, this settings is binded to it.